src/GeohashHilbert.jl

```diff
@@ -1,6 +1,6 @@
 module GeohashHilbert
 
-"sing Base.ImmutableDict in place of Dict for `ORDERING` and 
+"sing Base.ImmutableDict in place of Dict for `ORDERING` and
```

Comment:
I think you already fixed this line in the other PR, so it might be easier if you left this line unchanged in this PR to avoid conflicts when you merge them both. Same for all the whitespace changes in this PR.

Reply:
I'm not sure where those all came from. Some might be from converting Windows style line endings to Unix style, but that doesn't explain the deleted tabs. I'll remove what I can quickly, but given the small size of this PR I think it's probably faster to deal with a small merge conflict than hunt down every changed whitespace?

Reply:
Bigger yikes: looks like this file is mixed tabs and spaces (😱)

Reply:
Given all that, I think the lowest pain is just to ignore these for now, clean up the whitespace in the concurrent feature branch (#1 ), and endure a small merge conflict which will be resolved manually.